modules/op-installing-pipelines-operator-using-the-cli.adoc

```diff
@@ -27,7 +27,7 @@ spec:
   source: redhat-operators <3>
```

Comment:
For the module title, from:
`= Installing the {pipelines-shortname} Operator using the CLI`
to this (to erase double meaning)
`= Installing the {pipelines-shortname} Operator by using the CLI`